### PR TITLE
Update schema.json on 'make'

### PR DESCRIPTION
**Pull request by andrewkroh**
The schema.json was not being rebuilt when `make` was executed.

### CODE REVIEW

**Comment by webmat** [comment]
@andrew-goldstein @XavierM Hey guys, did you say this file was no longer in use?

If that's the case, I'd rather drop it completely. This is still based on the old file generation code, which doesn't include the nested fields.

File: Makefile [hunk 1]
```diff
@@ -94,10 +94,10 @@ misspell:
 .PHONY: reload_docs
 reload_docs: generator docs
 
-# Download and setup tooling dependencies.
-.PHONY: setup
-setup: ve
-	cd scripts && $(FORCE_GO_MODULES) go mod download
```
Comment (webmat) [inline Makefile]: Why are you deleting the `setup` target?

Comment (andrewkroh) [reply Makefile]: I think cut instead of copied. Fixed now.

**Comment by webmat** [comment]
Thanks for the fix!

Will probably delete it soon, as it's no longer being used by the SIEM team.

But I'll merge anyway for now. Perhaps we'll move it to legacy before deleting. I'm debating whether we should keep this legacy stuff. As Nic pointed out in the past, legacy is in the Github history too.

**Pull request merged**